hub/hub/doctype/hub_item_review/hub_item_review.py

```diff
@@ -7,4 +7,8 @@
 from frappe.model.document import Document
 
 class HubItemReview(Document):
-	pass
+
+    def before_naming(self):
+        if self.user:
+            user = frappe.get_doc('User', self.user)
+            self.username = user.full_name or user.username or None
```

Comment:
@rmehta @pratu16x7 What is a better way to achieve setting hub_item_review.username. I guess before_naming should only be used for setting `name`.

Reply:
A helper in the User doctype would do I feel, to `get_name()`